### PR TITLE
RHIROS-1077 Multiple values per query param

### PR DESCRIPTION
**Pull request by saltgen**
### List of supported parameters:

1. cluster
2. project
3. workload_type
4. workload
5. container

The idea is to have a logical OR between the different values of the respective query param, and filter the response accordingly.

 

### Sample request:

https:<url>?workload=servers&workload=Yuptoo-app

### CODE REVIEW

File: internal/model/recommendation_set.go [hunk 1]
```diff
@@ -48,13 +47,12 @@ func (r *RecommendationSet) GetRecommendationSets(orgID string, orderQuery strin
 			JOIN rh_accounts ON clusters.tenant_id = rh_accounts.id
 		`).Model(r).Preload("Workload.Cluster.RHAccount").Where("rh_accounts.org_id = ?", orgID)
```
Comment (patilsuraj767) [inline internal/model/recommendation_set.go:49]: Above query is returning incorrect results. We need to select `container_name` from recommendation_set table make a join on it. 
request you to change above query with the below.
```
query := db.Table("recommendation_sets").Joins(`
		JOIN (
			SELECT workload_id, container_name, MAX(monitoring_end_time) AS latest_monitoring_end_time 
			FROM recommendation_sets GROUP BY workload_id, container_name
		) latest_rs ON recommendation_sets.workload_id = latest_rs.workload_id 
				AND recommendation_sets.monitoring_end_time = latest_rs.latest_monitoring_end_time
				AND recommendation_sets.container_name = latest_rs.container_name
			JOIN workloads ON recommendation_sets.workload_id = workloads.id
			JOIN clusters ON workloads.cluster_id = clusters.id
			JOIN rh_accounts ON clusters.tenant_id = rh_accounts.id
		`).Model(r).Preload("Workload.Cluster.RHAccount").Where("rh_accounts.org_id = ?", orgID)
```

**Review by patilsuraj767 — CHANGES_REQUESTED** [review]

File: internal/model/recommendation_set.go (hunk 1 above)
Comment (saltgen) [reply internal/model/recommendation_set.go:49]: @patilsuraj767 Could you let me know about the issue in brief?

Comment (saltgen) [reply internal/model/recommendation_set.go:50]: Changes pushed

**Review by patilsuraj767 — APPROVED** [review]
LGTM :+1: Thanks @saltgen

**Pull request merged**